README.md

```diff
@@ -91,14 +89,36 @@ ics.createEvent({
 })
 ```
 
-3. Write multiple iCalendar files:
+3. Create multiple iCalendar events:
+```javascript
+const ics = require('ics')
+
+const events = ics.createEvents([
+  {
+    title: 'Lunch',
+    start: [2018, 1, 15, 12, 15],
+    duration: { minutes: 45 }
+  },
+  {
+    title: 'Dinner',
+    start: [2018, 1, 15, 12, 15],
+    duration: { hours: 1, minutes: 30 }
+  }
+])
+
+console.log(events)
+// [ { error: null,
+//     value: 'BEGIN:VCALENDAR\r\nVERSION:2.0\r\nCALSCALE:GREGORIAN\r\nPRODID:adamgibbons/ics\r\nBEGIN:VEVENT\r\nUID:0e71b0fd-c193-4c2d-a953-cf8a503b11c1\r\nSUMMARY:Lunch\r\nDTSTAMP:20180209T131000Z\r\nDTSTART:20180115T191500Z\r\nDURATION:PT45M\r\nEND:VEVENT\r\nEND:VCALENDAR\r\n' },
```

Comment:
BEGIN:VCALENDAR should not come twice for a single calendar feed, its fixed in the previous PR

Reply:
@adamgibbons, please, review us pull request, I added improvements to the @svnindia commit history for maintain the capability for actual version.